docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# Contributing Guidelines
+
+*We're so excited that you're interested in contributing to Neurobagel!* :partying_face:
+
+We appreciate all contributions, and hope that the below guidelines will make it as easy as possible for you to contribute to the [Neurobagel codebase](https://github.com/neurobagel) and to ensure your contribution can be easily integrated.
+
```

Comment:
Want to add a short bit on "You will need to make a GH account" here. I know that's basic, but I like the idea of making this a really low first bar. 

c.f. the very well written tedana contribution doc: https://github.com/ME-ICA/tedana/blob/main/CONTRIBUTING.md

Reply:
Sure. I actually have seen that contribution template haha - was trying to strike a balance between avoiding having our version be too verbose (most people probably won't read) and setting the bar for contributing too high.

In this case, I think adding the tip about the GH account makes sense.